scipy/stats/_continuous_distns.py

```diff
@@ -6123,6 +6146,21 @@ def _stats(self, b, moments='mv'):
     def _entropy(self, c):
         return 1 + 1.0/c - np.log(c)
 
+    def fit(self, data, *args, **kwds):
+        parameters = _check_fit_input_parameters(self, data, args, kwds)
+        data, fshape, floc, fscale = parameters
+        if floc is None:
+            raise RuntimeError("`floc` must be fixed to obtain a "
```

Comment:
```suggestion
            raise FitDataError("`floc` must be fixed to obtain a "
```
I think this fits in the same category of errors as all parameters being fixed and `floc` making the data out of range.

Reply:
I agree that `FitDataError` would be the best description for the error, but `FitDataError` shows this message: 

```
def __init__(self, distr, lower, upper):
        self.args = (
            "Invalid values in `data`.  Maximum likelihood "
            "estimation with {distr!r} requires that {lower!r} < x "
            "< {upper!r} for each x in `data`.".format(
                distr=distr, lower=lower, upper=upper),
        )
```

Which doesn't feel apt to convey that the user needs to set a `floc`.

Reply:
I could bootstrap the existing message to say: 


```
FitDataError: Invalid values in `data`.  Maximum likelihood estimation with
'pareto' requires that '`floc` be fixed to obtain a well defined solution and
 0 < x < inf for each x in `data`.
```
by writing `lower="floc be fixed to obtain a well defined solution and 0"`.